node/manifest.ts

```diff
@@ -43,6 +43,17 @@ interface GenerateManifestOptions {
   layers?: Layer[]
 }
 
+const removeEmptyConfigValues = (functionConfig: EdgeFunctionConfig) =>
```

Comment:
Why do we need to remove empty values?

Reply:
I did this because the logic before this would allow for undefined values to be passed. For example. if we had `on_error` but `functionConfig.excluded_patterns = []` we would otherwise still get that empty array added, or fields that were undefined. Like `generator: undefined`. Essentially I tried to optimise [this line](https://github.com/netlify/edge-bundler/pull/348/files#diff-e659ded745234fd55b239ea6728eef800523583941fd1542d88b44373cd39e8aL56) so I wouldn't have to add a bunch of other config values. And even if I did, if one of them was true, it would add undefined values for the others.